front-end/src/components/Comment/Comment.tsx

```diff
@@ -41,22 +43,36 @@ interface Props{
 		Promise<ApolloQueryResult<DiscussionPostAndCommentsQuery>>
 }
 
-export const Comment = ({ className, comment, refetch } : Props) => {
+export const Comment = ({ className, comment, last, refetch } : Props) => {
 	const { author, content, created_at, id, updated_at } = comment;
+	const { hash } = useLocation();
+
+	useEffect(() => {
+		if (hash === `#${id}` || hash === '#end') {
+			setTimeout(
+				() => {
+					const element = document.getElementById(id);
```

Comment:
I think this is not how it should be done in React. https://stackoverflow.com/questions/38093760/how-to-access-a-dom-element-in-react-what-is-the-equilvalent-of-document-getele/38093981#38093981